### ISSUE TITLE
NTR: alphabeta-crystallin high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8449]
New term request alphabeta-crystallin high in CSF (segal_080402173425): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000399
- label: Elevated CSF alpha-crystallin B chain concentration
- parents: [Abnormal CSF protein concentration (HP:0025456)](https://hpo.jax.org/app/browse/term/HP:0025456)
- pmid: 
- def: The concentration of alpha-crystallin B chain in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8449
- synonyms: Alphabeta-crystallin high in CSF(NONE)
